content/en/community/hackathons/sessions/baby-steps/content/Setup_Helloworld.md

```diff
@@ -0,0 +1,287 @@
+It is **heavily** recommended that for building and developing applications with Unikraft, you use a conventional folder structure.
+Namely, for the examples that we will be going through in this session, our working directory will look like this:
+
+```console
+workdir                     ---> working directory containing applications
+`-- helloworld              ---> application directory
+    |
+    |
+    |-- main.c              ---> application code
+    |-- Makefile            ---> makefile for specifying external library dependencies
+    |-- Config.uk           ---> unikraft configuration options for the application
+    |-- Makefile.uk         ---> the application's compiler flags and source files to build
+    |
+    |
+    `-- .unikraft
+        |-- libs        ---> external libraries used by the application
+        `-- unikraft    ---> unikraft core kernel, containing all the internal libraries
+`-- nginx
+    |
+    |
+    |
+`-- python3
+    |
+    |
+    |
+...
+```
+
+Let's begin with the simplest app there is, `helloworld`.
+We first create the `workdir` folder where we will place all of our applications and clone the [`app-helloworld`](https://github.com/unikraft/app-helloworld/) repository, which contains the source files and the Makefiles necessary for bulding the `helloworld` application.
+
+```bash
+$ mkdir workdir
+
+$ cd workdir/
+
+$ git clone https://github.com/unikraft/app-helloworld helloworld
+
+$ cd helloworld/
+[...]
+
+$ ls -F
+[...]  Makefile  Makefile.uk  README.md  main.c  run-qemu-x86_64.sh* [...]
+```
+
+Now, while inside the `helloworld/` directory, we create the `.unikraft/` folder, which will store the Unikraft core kernel from this [repository](https://github.com/unikraft/unikraft), along with the `libs/` directory which should store all of the external libraries the applications depends on, cloned from the `lib-` repos of the [Unikraft organization](https://github.com/unikraft/).
+Since the `helloworld` application is fairly simple, it does not use any external libray, so we can leave the `libs/` directory empty.
```

Comment:
```suggestion
Since the `helloworld` application is fairly simple, it does not use any external library, so we can leave the `libs/` directory empty.
```